flink-core/pom.xml

```diff
@@ -70,6 +74,18 @@ under the License.
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
```

Comment:
We don't need the JMH dependencies in flink-core anymore, because the benchmarks have been move to the flink-benchmark module, right?